backend/src/agent/persistence/main.go

```diff
@@ -108,7 +110,7 @@ func main() {
 	go swfInformerFactory.Start(stopCh)
 	go workflowInformerFactory.Start(stopCh)
 
-	if err = controller.Run(2, stopCh); err != nil {
+	if err = controller.Run(numWorker, stopCh); err != nil {
```

Comment:
Do we need some sanity check on the value? E.g., what if we pass a negative value to controller.Run as numWorker

Reply:
I think the server will just fail with an error message, then the user who configured it should look at that.